test/solc/CommandLineParser.cpp

```diff
@@ -175,7 +175,7 @@ BOOST_AUTO_TEST_CASE(cli_mode_options)
 		expectedOptions.output.dir = "/tmp/out";
 		expectedOptions.output.overwriteFiles = true;
 		expectedOptions.output.evmVersion = EVMVersion::spuriousDragon();
-		expectedOptions.output.experimentalViaIR = true;
+		expectedOptions.output.viaIR = true;
```

Comment:
Also, I'd add a separate small test in this file that checks that both `--experiemental-via-ir` and `--ir` on their own set `expectedOptions.output.viaIR` to true. You can use this one as a base - just remove most of the flags and most of the expectations (they should stay at defaults without the flags).